es-app/src/components/TextListComponent.h

```diff
@@ -30,7 +30,7 @@ class TextListComponent : public IList<TextListData, T>
 	using IList<TextListData, T>::getTransform;
 	using IList<TextListData, T>::mSize;
 	using IList<TextListData, T>::mCursor;
-	using IList<TextListData, T>::Entry;
```

Comment:
Compilation error:

In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/
gamelist/BasicGameListView.cpp:1:
In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/
gamelist/BasicGameListView.h:5:
/home/devel/temp/EmulationStation_Stable/es-app/src/components/TextListComponent
.h:33:32: error: dependent using declaration resolved to type without 'typename'
        using IList<TextListData, T>::Entry;
                                      ^